addons/web_editor/static/lib/odoo-editor/src/powerbox/Powerbox.js

```diff
@@ -174,8 +175,11 @@ export class Powerbox {
         this.options.onActivate && this.options.onActivate();
         this._currentOpenOptions = openOptions;
 
+        const selection = this.options.document.getSelection();
         const openOnKeyupTarget =
             this._currentOpenOptions.openOnKeyupTarget || this.options.editable;
+        const currentNode =
+            this._currentOpenOptions.currentNode || this.options.editable;
```

Comment:
@deso-odoo declare them where its needed, in your case it should be above `showOnceOnKeyup` as you started using selection, cuurentNode after this function.Its just a good practice to keep in mind. Cheers!